public_html/wp-content/mu-plugins/blocks/source/components/item-select/style.scss

```diff
@@ -24,6 +24,10 @@
 	align-items: center;
```

Comment:
Oh, I forgot to mention earlier, but it looks like the commit message needs a trailing `.` on the short description, and `Fixes #81`. If the short description can fit the _why_ of the commit, in addition to the _what_, that'd be good to. e.g., `Increase text contrast to improve a11y.`

Reply:
Yep, once I actually write the merge commit, I'll add this in.